src/spatialNavigation.js

```diff
@@ -385,22 +391,28 @@ class SpatialNavigation {
           this.logIndex += 1;
         }
 
-        const eventType = findKey(this.getKeyMap(), (code) => event.keyCode === code);
+        const eventType = this.getEventType(event.keyCode);
 
         if (!eventType) {
           return;
         }
 
+        this.pressedKeys[eventType] = this.pressedKeys[eventType] ? this.pressedKeys[eventType] + 1 : 1;
+
         event.preventDefault();
         event.stopPropagation();
 
+        const details = {
+          pressedKeys: this.pressedKeys
+        };
+
         if (eventType === KEY_ENTER && this.focusKey) {
-          this.onEnterPress();
+          this.onEnterPress(details);
```

Comment:
As far as I understand here you need to pass some data to the callback that you will use at your project, to perform some specific action on the (enter) key.
Isn't better to only pass event or eventType here, and then resolve the enter logic in the specific project ?

Reply:
But we have this special Ok/Enter press handling everywhere, so it is fine for it to be a “special” callback. Now it will get item props and details additionally as 2nd param

Reply:
all right then 👍